app/models/moirai/translation.rb

```diff
@@ -3,5 +3,15 @@
 module Moirai
   class Translation < Moirai::ApplicationRecord
     validates_presence_of :key, :locale
+
+    def find_file_path
+      @key_finder ||= KeyFinder.new
+      @key_finder.file_path_for(key, locale: locale)
+    end
+
+    def self.by_file_path(file_path)
+      key_finder = KeyFinder.new
+      all.select { |translation| key_finder.file_path_for(translation.key, locale: translation.locale) == file_path }
+    end
```

Comment:
same for this

Reply:
but they are not already :) 
I think it makes more sense to have KeyFinder depend on Translation, than the other way around.

Reply:
I'd rather refactor this in a separate PR 👍